header_test.sh

```diff
@@ -0,0 +1,19 @@
+# !/usr/bin/env bash
+
+# Collate files
+python_files="$(find ./cli ./smarts ./benchmark ./envision -name '*.py')"
+js_files="$(find ./envision/web/src -name '*.js')"
```

Comment:
used `./envision/web/src` since all js files are in src directory. and getting to the src level helps to avoid checking in web/node_modules directory